sql-native/src/main/java/org/ballerinalang/sql/nativeimpl/ClientProcessor.java

```diff
@@ -65,32 +53,14 @@ public static Object close(BObject client) {
         return null;
     }
 
-    private static SQLDatasource.SQLDatasourceParams createSQLDatasourceParams
-            (BMap<BString, Object> sqlDatasourceParams, BMap<BString, Object> globalConnectionPool) {
-        BMap<BString, Object> connPoolProps = (BMap<BString, Object>) sqlDatasourceParams
-                .getMapValue(Constants.SQLParamsFields.CONNECTION_POOL_OPTIONS);
-        Properties poolProperties = null;
-        if (connPoolProps != null) {
-            poolProperties = new Properties();
-            for (BString key : connPoolProps.getKeys()) {
-                poolProperties.setProperty(key.getValue(), connPoolProps.getStringValue(key).getValue());
-            }
+    public static Object createClient(BObject client, SQLDatasource.SQLDatasourceParams sqlDatasourceParams) {
```

Comment:
Lets move the logic to the calling method itself and no need for a  separate method as this is the only usage of this method.

Reply:
`CreateSqlClient` method is only use for sql test to create a sql client. All the connector modules use `createClient`. Two methods have different parameters. If we move the `createClient` into `createSqlClient` method we have to change `mysql` and `jdbc` modules.
If we use `createClient` instead of `createSqlClient` we loose the ability to assign parameters separately for connector modules.

Reply:
@SasinduDilshara IMO its fine if you have to migrate JDBC & MySQL modules, this will be more confusing. What is the usecase the clients needs to overide this method? AFAIK this only sets the params passed from the client.

Reply:
Add ClienTestUtil Class in test-util module.
Commit id is 1fec1bb22a6a9c3f3ff10e0aac1a0a84b9a360cd